01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,13 @@
+21.times do |x|
```

Comment:
```suggestion
1.upto(20) do |x|
```

とすれば`next if x==0`がなくせます。

あと、ブロックパラメータはintegerのiかnumberのnを使うことが多いですね。

```suggestion
1.upto(20) do |n|
```

Reply:
今回の場合、コメント頂いた`1.upto(20) do |i|`以外では、`for i in 1..20 do`でも`next if i == 0`をなくせるので問題ないという認識で合ってますか？
また、どちらの方がより適しているなどはありますか？

Reply:
forは慣習としてRubyではほとんど使われないですね。
uptoの方がRubyらしいコードだと思います！